libs/platform/src/lib/components/list/base-list-item.ts

```diff
@@ -14,7 +14,7 @@ import { BaseComponent } from '../base';
 import { ListConfig } from './list.config';
 import { ActionListItemComponent } from './action-list-item/action-list-item.component';
 
-const isActive = 'is-active';
+export const ISACTIVE = 'is-active';
```

Comment:
Can you make this `IS_ACTIVE`?

Reply:
I would recommend using even `IS_ACTIVE_CLASS`

Reply:
Did changes as @JKMarkowski as suggested , so please re-review it.